Assets/Scripts/Models/Area/RoomManager.cs

```diff
@@ -0,0 +1,429 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace ProjectPorcupine
```

Comment:
I prefer `ProjectPorcupine.Rooms` too, although that is not the folder name... it might be good to make a separate folder? not sure here.